docs/setuptools.txt

```diff
@@ -107,6 +110,21 @@ As you can see, it doesn't take much to use setuptools in a project.
 Run that script in your project folder, alongside the Python packages
 you have developed.
 
+For Python 3.3+, and whenever you are using PEP 420 compliant implicit
```

Comment:
I don't think this should be under "Basic Use". The majority of people will *not* be using namespace packages, and we should probably go into more detail about the exact caveats of using this.

Reply:
Hm, where should we put it then? As for exact caveats, what do you have in mind? Present the user with a possible project structure and folders underneath some source root that look like packages but should be excluded?

Reply:
I will put it into its own section directly below Using find_packages().